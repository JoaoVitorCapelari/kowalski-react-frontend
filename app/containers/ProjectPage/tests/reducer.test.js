import projectPageReducer from '../reducer';

describe('projectPageReducer', () => {
  it('returns the initial state', () => {
    expect(projectPageReducer(undefined, {})).toMatchSnapshot();
  });
});
