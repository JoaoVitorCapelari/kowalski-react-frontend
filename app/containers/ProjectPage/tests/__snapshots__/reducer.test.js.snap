// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`projectPageReducer returns the initial state 1`] = `
Immutable.Map {
  updateProjectAttributesErrorMsg: "",
  users: Immutable.List [
  ],
  activityFilteringText: "",
  loadingProjectError: "",
  projectLoadedIntoNewTaskForm: null,
  projectCodes: Immutable.List [
  ],
  expandedActivityIds: Immutable.List [
  ],
  isNewTaskFormDialogOpened: false,
  isAddPeopleFormOpen: false,
  activityLoadedIntoNewTaskForm: null,
  isSubmittingNewTask: false,
  projectLoadedIntoNewActivityForm: null,
  selectedProjectId: null,
  updateProjectAttributesStatus: Immutable.Map {
  },
  isNewActivityFormDialogOpened: false,
  selectedTab: 0,
  loadingUsersError: "",
  inlineProjectFormFields: Immutable.Map {
    name: Immutable.Map {
      state: "STATE_NORMAL",
      value: undefined,
      originalValue: undefined,
    },
    accountableId: Immutable.Map {
      state: "STATE_NORMAL",
      value: undefined,
      originalValue: undefined,
    },
    startDate: Immutable.Map {
      state: "STATE_NORMAL",
      value: undefined,
      originalValue: undefined,
    },
    endDate: Immutable.Map {
      state: "STATE_NORMAL",
      value: undefined,
      originalValue: undefined,
    },
    description: Immutable.Map {
      state: "STATE_NORMAL",
      value: undefined,
      originalValue: undefined,
    },
    code: Immutable.Map {
      state: "STATE_NORMAL",
      value: undefined,
      originalValue: undefined,
    },
  },
  inlineProjectFormErrorMsg: "",
  selectedProject: null,
  isSubmittingNewActivity: false,
  loadingProjectCodesError: "",
}
`;
